### PR TITLE
small fix

### PR DESCRIPTION
**Pull request by PetarT**

### CODE REVIEW

File: libraries/redcore/layouts/toolbar/button/link.php [hunk 1]
```diff
@@ -26,7 +26,14 @@
 $text = $button->getText();
 
 // Get the button class.
-$btnClass = 'btn btn-default';
+if ($isOption)
```
Comment (Kixo) [inline libraries/redcore/layouts/toolbar/button/link.php]: $btnClass = $isOption ? '' : 'btn btn-default';
much smaller

Comment (PetarT) [inline libraries/redcore/layouts/toolbar/button/link.php]: will do :+1:

File: libraries/redcore/toolbar/builder.php [hunk 2]
```diff
@@ -47,21 +47,24 @@ public static function createCopyButton($task, $class = '')
...
-	public static function createNewButton($task, $class = '')
+	public static function createNewButton($task, $class = '', $isOption = false)
 	{
-		if (empty($class))
-		{
-			$class = 'btn-success';
-		}
-
-		else
+		if (!$isOption)
```
Comment (Kixo) [inline libraries/redcore/toolbar/builder.php]: hmm....
I would like to be able to add a class to the links if needed
so why not something like this
if (empty($class) && !$isOption)
		{
			$class = 'btn-success';
		}
		else
		{
			$class .= ' btn-success';
		}

Comment (PetarT) [inline libraries/redcore/toolbar/builder.php]: Yup, you are right. I intend to make it that way, not sure why I had brain fart there

File: libraries/redcore/toolbar/builder.php [hunk 3]
```diff
@@ -194,18 +194,18 @@ public static function createCancelButton($task, $class = '')
...
-	 * @param   string  $task   The task name.
-	 * @param   string  $class  A css class to add to the button.
+	 * @param   string  $task      The task name.
+	 * @param   string  $class     A css class to add to the button.
+	 * @param   bool    $isOption  Is button a menu option?
 	 *
 	 * @return  RToolbarButtonStandard  The button.
 	 */
-	public static function createCloseButton($task, $class = '')
+	public static function createCloseButton($task, $class = '', $isOption = false)
```
Comment (Kixo) [inline libraries/redcore/toolbar/builder.php]: one idea...

what if we make it like this...



	/**
	 * Create a new button.
	 *
	 * @param   string  $task        The task name.
	 * @param   string  $class       A css class to add to the button.
	 * @param   string  $classAddon  A css class add-on.
	 *
	 * @return  RToolbarButtonStandard  The button.
	 */
	public static function createNewButton($task, $class = '', $classAddon = 'btn-success')
	{
		$class = !empty($class) ? $class . ' ' . $classAddon : $classAddon;

		return new RToolbarButtonStandard('JTOOLBAR_NEW', $task, $class, 'icon-file-text', '', false);
	}

File: libraries/redcore/toolbar/builder.php [hunk 4]
```diff
@@ -219,16 +219,16 @@ public static function createCloseButton($task, $class = '')
 	 *
 	 * @param   string  $task   The task name.
 	 * @param   string  $class  A css class to add to the button.
+	 * @param   bool    $isOption  Is button a menu option?
```
Comment (Kixo) [inline libraries/redcore/toolbar/builder.php]: travis is failing because this is not aligned

**Pull request merged**